### PR TITLE
feat: navigate to network

### PR DESCRIPTION
**Pull request by dhemutton**
This PR contains the navigation to verify page based on the config network.

### CODE REVIEW

File: src/components/NavigationBar/NavigationBar.test.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        <Route exact path="/">
+          <NavigationBar logout={() => {}} />
+        </Route>
+      </BrowserRouter>
+    );
+
+    expect(screen.getByText("Verify Documents").closest("a")).toHaveAttribute(
+      "href",
+      "https://tradetrust.io/#verify-documents"
+    );
```
Comment (simboonlong) [inline src/components/NavigationBar/NavigationBar.test.tsx:42]: so it's `queryByText` will do:

```
expect(screen.queryByText("Verify Documents")).toHaveAttribute(
  "href",
  "https://tradetrust.io/#verify-documents"
);
```

File: src/components/NavigationBar/NavigationBar.test.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+    expect(screen.queryAllByText("Verify Documents")).not.toBeNull();
+    expect(screen.queryAllByText("Create Documents")).not.toBeNull();
+    expect(screen.queryAllByText("Resources")).not.toBeNull();
+    expect(screen.queryAllByText("FAQ")).not.toBeNull();
+    expect(screen.queryAllByText("Contact")).not.toBeNull();
+    expect(screen.queryByTestId("settings-icon")).not.toBeNull();
+  });
+
+  it("should go to default verify documents page without config", () => {
```
Comment (simboonlong) [inline src/components/NavigationBar/NavigationBar.test.tsx:30]:
```suggestion
  it("should render href to default verify documents page without config", () => {
```

File: src/components/NavigationBar/NavigationBar.test.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import { NavigationBar } from "./NavigationBar";
+import { BrowserRouter, Route } from "react-router-dom";
+import rinkebyConfig from "../../test/fixtures/sample-rinkeby-config.json";
+import ropstenConfig from "../../test/fixtures/sample-config.json";
+
+const mockMainnet = {
```
Comment (simboonlong) [inline src/components/NavigationBar/NavigationBar.test.tsx:8]:
```suggestion
const mockMainnetConfig = {
```

File: src/components/NavigationBar/NavigationBar.tsx [hunk 4]
```diff
@@ -2,14 +2,15 @@ import React, { FunctionComponent } from "react";
 import { Link } from "react-router-dom";
 import { Settings } from "react-feather";
 import logo from "./logo.svg";
+import { usePersistedConfigFile } from "../../common/hook/usePersistedConfigFile";
 
-const urlPath = "https://tradetrust.io";
+const homesteadUrlPath = "https://tradetrust.io";
 
 const navItems = [
```
Comment (simboonlong) [inline src/components/NavigationBar/NavigationBar.tsx:9]: wanna typecheck this?

File: src/components/NavigationBar/NavigationBar.tsx [hunk 5]
```diff
@@ -19,30 +20,41 @@ const navItems = [
...
   },
   {
     id: "settings",
     label: "Settings",
-    path: urlPath + "/settings",
+    path: homesteadUrlPath + "/settings",
   },
 ];
 
+const getVerifyPath = (network: string): string => {
```
Comment (simboonlong) [inline src/components/NavigationBar/NavigationBar.tsx:42]: typecheck this as `"homestead" | "ropsten" | "rinkeby"`

File: src/components/NavigationBar/NavigationBar.test.tsx (hunk 1 above)
Comment (dhemutton) [reply src/components/NavigationBar/NavigationBar.test.tsx:42]: Based on: [this link](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence) might be better to keep as `getByText` for an error to be thrown if it does not exist.

Screenshot 2020-11-02 at 5 02 11 PM

Comment (simboonlong) [reply src/components/NavigationBar/NavigationBar.test.tsx:42]: ah nice, then how about:

```
expect(screen.getByText("Verify Documents").getAttribute("href")).toBe(
  "https://tradetrust.io/#verify-documents"
);
```

your solution works fine actually, just that i feel maybe can try to omit `.closest("a")`, cos there is an assumption/opinion on html markup structure when it is doing a find `closest` anchor tag, we'll want to keep it as less opinionated as possible

**Review by simboonlong — APPROVED** [review]
lgtm

**Pull request merged**